### ISSUE TITLE
No mapping of stylistic set names

### ISSUE THREAD

**Issue by hyvyys** [opentypejs/opentype.js#399]
## Expected Behavior
For a font where OpenType stylistic sets have assigned human readable names, they are contained in the font object in `tables.names`, at indices starting from `256`. Stylistic sets are contained in `tables.gsub.features` where each entry has format `{ tag, feature { featureParams, lookupListIndexes} }`. I would expect a reference to the index of the corresponding entry from `tables.names` somewhere in there.

## Current Behavior
There is no information on which name record belongs to which stylistic set. Technically it is possible to guess that the stylistic set names are stored consecutively but this assumption fails if not all stylistic sets have assigned names, e.g. a font could contain names only for ss01, ss02, ss03, and, why not, ss20. Then the name for ss20 would be wrongly assigned to ss04, and for further sets other metadata like copyright notice, designer name, font names which are also stored in `tables.names` could be wrongly assigned if we don't take precautions.

## Possible Solution
Add index of the name record to the feature object.

## Context
I am building a GUI for manual testing of fonts at https://hyvyys.github.io/Bulletproof/

## My Environment
* Version used: 1.1.0
* Font used: custom
* Link to your project: https://hyvyys.github.io/Bulletproof/

**Comment by hyvyys** [comment]
Update: actually the feature names are (obviously) not always stored starting at name id 256, their location is defined in the GSUB table. Example output from fontTool's TTX module (the font is Bungee):

```
      <FeatureRecord index="26">
        <FeatureTag value="ss02"/>
        <Feature>
          <FeatureParamsStylisticSet>
            <Version value="0"/>
            <UINameID value="256"/>  <!-- Round forms -->
          </FeatureParamsStylisticSet>
          <!-- LookupCount=6 -->
          <LookupListIndex index="0" value="5"/>
          <LookupListIndex index="1" value="6"/>
          <LookupListIndex index="2" value="7"/>
          <LookupListIndex index="3" value="8"/>
          <LookupListIndex index="4" value="9"/>
          <LookupListIndex index="5" value="10"/>
        </Feature>
      </FeatureRecord>
```

**Comment by hyvyys** [comment]
Currently the offset to FeatureParams table seems to be extracted, e.g.

```
{
  "tag": "ss01",
  "feature": {
    "featureParams": 6,
    "lookupListIndexes": [
      22
    ]
  }
}
```

However, I have no idea where that FeatureParams table is, where the offset is in respect to, and how to wrap my head around this. I'd be happy to contribute but I'd need some noob-level pointers.

**Comment by AndreasFaust** [comment]
Is there a solution yet?

**Comment by Connum** [comment]
> Is there a solution yet?

Yes, see the linked pull request, which requires conflict resolving and some other care (pinging @ILOVEPIE :))

**Comment by AndreasFaust** [comment]
Thank you, @Connum ! I did not look close enough. 
This feature would be amazing. My approach to circumvent it is writing the name of the feature into the `UI Name ID` to be able to find it.

**Comment by ILOVEPIE** [comment]
> > Is there a solution yet?
> 
> Yes, see the linked pull request, which requires conflict resolving and some other care (pinging @ILOVEPIE :))

Hey, I got your ping the other day late at night, so I wasn't able to really take a good look at it. I'm sorry that I haven't been too active. I've been really busy with work and some other stuff. There's a lot going on in my life at the moment. But I will take a look at it. Can you send a direct link to PR? Because I can't seem to find it here in the GitHub app.

**Comment by Connum** [comment]
> Can you send a direct link to PR? Because I can't seem to find it here in the GitHub app.

Sure, it's #452
